origami.json

```diff
@@ -6,6 +6,15 @@
 		"email": "origami.support@ft.com",
 		"slack": "financialtimes/origami-support"
 	},
+	"browserFeatures": {
+		"required": [
+			"Array.from",
+			"Array.prototype.includes",
+			"IntersectionObserver",
```

Comment:
Lee said: If we make the feature only run if `IntersectionObserver` exists then this would not be a breaking change